### DISCUSSION TITLE
Pull repo with hidden user and password

### DISCUSSION THREAD

**Discussion by ramazulay** [D10S0VSkY-OSS/Stack-Lifecycle-Deployment#72] [Ideas]
Hi @D10S0VSkY-OSS,
From your DOC you recommended to pass the user&password like the example: https://username:password@github.com/aws_vpc.
But in the UI it is **not hidden**, what do you suggest doing?(over https)

Thanks.

**Comment by D10S0VSkY-OSS** [comment]
Hi @ramazulay 
the best way is that you manage the secrets from kubernetes, in case you use ssh or access token...
